### PR TITLE
Add Black to our Python standards

### PR DESCRIPTION
**Pull request by willbarton**
We've started using Black in [various](https://github.com/cfpb/django-flags) [projects](https://github.com/cfpb/retirement) and it has been helpful as we need to add linting to projects that predate our linting standards.

This change adds documentation about how we're using Black and clarifies that we've deviating from Black's preferences on line length (because we're all set up for our standard PEP8 79 char).

### CODE REVIEW

File: standards/python.md [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
   - [Specifying dependencies in `requirements.txt` or `setup.py`](#specifying-dependencies-in-requirementstxt-or-setuppy)
 
-## Linting
+## Style
 
 All of our Python code should match the [PEP8 style guide](https://www.python.org/dev/peps/pep-0008/) and the [Django coding style guidelines](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/).
 
 We follow [PEP8's line length](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) instead of Django's for consistency with our other development standards. We follow [Django's recommended import ordering in Django projects](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#imports).
 
+We use [Black](http://black.readthedocs.io) to standardize formatting of our Python code and make linting easier. We do override the default line length for Black, however, to 78. We also exclude generated files like migrations. We provide a sample [`pyproject.toml` file containing line length and exclude configuration for Black](../pyproject.toml).
```
Comment (chosak) [inline standards/python.md]:
```suggestion
We use [Black](http://black.readthedocs.io) to standardize formatting of our Python code and make linting easier. We override the default Black line length, setting it to 79 to match PEP8. We also exclude generated files like migrations. We provide a sample [`pyproject.toml` file containing line length and exclude configuration for Black](../pyproject.toml).
```

**Comment by Scotchester** [comment]
Just to ask one more time before we lock this initial recommendation in: Are we _sure_ we don't want to consider the Black default line length?

**Comment by willbarton** [comment]
> Just to ask one more time before we lock this initial recommendation in: Are we sure we don't want to consider the Black default line length?

I don't, personally. I'd be open to considering it, but that seems far more disruptive to our daily work than just black itself.

**Comment by higs4281** [comment]
No strong objection to Black's default, but I vote to stick with PEP8.

**Pull request merged**